src/components/Task.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div id="task">
+   <h1>ToDo リスト</h1>
+
+    <input type="radio" name="condition" value="すべて" v-model="picked" checked>すべて
+    <input type="radio" name="condition" value="作業中" v-model="picked">作業中
+    <input type="radio" name="condition" value="完了" v-model="picked">完了
+
+    <table :class="table">
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>コメント</th>
+          <th>状態</th>
+        </tr>
+      </thead>
+      <tbody v-for="todos in filterStatus" :key="todos.id">
+        <tr>
+          <td>{{todos.id}}</td>
+          <td>{{todos.content}}</td>
+          <td>
+            <button @click="changeStatus(todos.id)">{{todos.status}}</button>
+            <button @click="removeTodo(todos.id)">削除</button>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    <h2>新規タスクの追加</h2>
+    <input type="text" name="newTask" v-model="text">
+    <button @click="addTodo">追加</button>
+  </div>
+</template>
+
+<script>
+export default{
+  data: function() {

```

Comment:
以下のように省略して記述することが多いです。
```suggestion
  data() {
```
computedやmethodsの関数も同様にできますので、修正お願いします。